kubernetes-maven-plugin/doc/src/main/asciidoc/inc/_introduction.adoc

```diff
@@ -12,22 +18,22 @@ For the full power, in order to tune all facets of the creation, external _resou
 [[building-images]]
 == Building Images
 
-The <<k8s:build>> goal is for creating Docker images containing the actual application. These then can be deployed later on Kubernetes or OpenShift.
+The <<jkube:build>> goal is for creating Docker images containing the actual application. These then can be deployed later on Kubernetes or OpenShift.
```

Comment:
Umm, maybe we should create one parameter for the plugin prefix also. WDYT?

Reply:
oh, just noticed there is `{goal-prefix}` . I think `{goal-prefix}` should be there in place of `jkube`

Reply:
https://github.com/eclipse/jkube/pull/177#discussion_r420042628